Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -31,7 +31,7 @@
     <Copyright>Copyright Dennis Doomen 2010-2020</Copyright>
     <LangVersion>9.0</LangVersion>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+  <PropertyGroup Condition="'$(TargetFramework)' != 'net6.0'">
```

Comment:
Noticed that I had messed up the operator.
Without adding net6.0 this fix increased the build time on my machine from 1.7s to 5.0s.